examples/jenkins/application-template.json

```diff
@@ -314,7 +349,27 @@
     }
   ],
   "parameters": [
+     {
+      "name": "DEPLOYMENT_REQUEST_SIZE",
+      "description": "Memory rquested on startup from OpenShift Quota System",
+      "value": "64Mi"
+    },
     {
+      "name": "DEPLOYMENT_LIMIT_SIZE",
+      "description": "Maximum amount of memory allowed by OpenShift Quota System",
+      "value": "128Mi"
+    },
+     {
+      "name": "BUILD_REQUEST_SIZE",
+      "description": "Memory rquested on startup from OpenShift Quota System",
+      "value": "32Mi"
+    },
+    {
+      "name": "BUILD_LIMIT_SIZE",
```

Comment:
The online beta where limit or quota wasn't defined is a bug that will be
addressed by project quota defaults.  So we shouldn't require it in
general, and we should only *NEED* to include it in the template if it's
outside the reasonable default to execution.  We want build requests to be
best effort as much as possible.

On Thu, Jan 28, 2016 at 10:48 AM, Gabe Montero <notifications@github.com>
wrote:

> In examples/jenkins/application-template.json
> <https://github.com/openshift/origin/pull/6858#discussion_r51139771>:
>
> >      {
> > +      "name": "DEPLOYMENT_LIMIT_SIZE",
> > +      "description": "Maximum amount of memory allowed by OpenShift Quota System",
> > +      "value": "128Mi"
> > +    },
> > +     {
> > +      "name": "BUILD_REQUEST_SIZE",
> > +      "description": "Memory rquested on startup from OpenShift Quota System",
> > +      "value": "32Mi"
> > +    },
> > +    {
> > +      "name": "BUILD_LIMIT_SIZE",
>
> In my testing for the online beta, where a quota was defined, the pod
> would not be started (with an explicit message stating such) unless a limit
> or request was defined. That coupled with the idea of updating the origin
> templates to service the beta given to me lead to his PR.
>
> I'll switch the build resource element to a request.
>
> I'm assuming that BUILD_LIMIT_SIZE is not liked as well. I'll go with
> "MEMORY_REQUEST" (in line with MEMORY_LIMIT, unless that is not liked),
> update the description with the implications to the user pulled from the
> doc, etc.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6858/files#r51139771>.
>